src/telemetry/stats.c

```diff
@@ -538,27 +547,25 @@ ts_telemetry_stats_gather(TelemetryStats *stats)
 		{
 			case RELTYPE_HYPERTABLE:
 				Assert(NULL != ht);
-				process_hypertable(&stats->hypertables.storage.base, class, ht);
+				process_hypertable(&stats->hypertables.ss.bs, class, ht);
 				break;
 			case RELTYPE_DISTRIBUTED_HYPERTABLE:
 				Assert(NULL != ht);
-				process_distributed_hypertable(&stats->distributed_hypertables.storage.base,
-											   class,
-											   ht);
+				process_distributed_hypertable(&stats->distributed_hypertables.ss.bs, class, ht);
```

Comment:
This is unnecessarily complicated and just opens up the window for run-time bugs that could have been caught at compile-time.
```suggestion
				process_distributed_hypertable(&stats->distributed_hypertables, class, ht);
```